### ISSUE TITLE
Check all / Uncheck all links not accessible through keyboard navigation

### ISSUE THREAD

**Issue by javoparada** [ehynds/jquery-ui-multiselect-widget#867]
### What is the current behavior?

Currently, if I press the TAB key focusing on the dropdown when it's opened, it will close it, ignoring both check all and uncheck all buttons.

### What is the expected behavior?

When you press the TAB key with the dropdown open it will focus on the check all / uncheck all buttons at some point (before or after the options).

### Steps to reproduce

- Open the dropdown
- Press the Tab Key.

**Comment by javoparada** [comment]
I've fixed this by adding the missing href attribute to each <a> element created.

I hope you include this in your next release,

Thanks!
